components/mediation/data-publishers/org.wso2.micro.integrator.prometheus.observability/src/main/java/org/wso2/micro/integrator/prometheus/observability/metrics/util/PrometheusMetricsConstants.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.micro.integrator.prometheus.observability.metrics.util;
+
+/**
+ * Constants required in instrumenting Prometheus metrics.
+ */
+public class PrometheusMetricsConstants {
+
+   // public static final String PROXY_NAME = "proxy.name";
+    public static final String SYNAPSE_REST_API = "SYNAPSE_REST_API";
+    public static final String REMOTE_HOST = "REMOTE_HOST";
+    public static final String INBOUND_ENDPOINT = "inbound_endpoint";
+    public static final String INBOUND_ENDPOINT_NAME = "inbound.endpoint.name";
+
+    public static final String PROXY_LATENCY_TIMER = "PROXY_LATENCY_TIMER";
+    public static final String API_LATENCY_TIMER = "API_LATENCY_TIMER";
+    public static final String INBOUND_ENDPOINT_LATENCY_TIMER = "INBOUND_ENDPOINT_LATENCY_TIMER";
+
+    public static final String HTTP_PORT = "http.nio.port";
+    public static final String JAVA_VERSION = "java.vm.specification.version";
+    public static final String JAVA_HOME = "java.home";
+    public static String PORT_OFFSET = "portOffset";
+    public static String INTERNAL_HTTP_API_PORT = "internal.http.api.port";
```

Comment:
can we make these two variables also final?

Reply:
Fixed